src/tests-rosetta/api.ts

```diff
@@ -3289,6 +3290,106 @@ describe('Rosetta API', () => {
     );
   })
 
+  test('block/transaction coinbase', async () => {
+    const dbBlock = await db.getCurrentBlock();
+    const blockRes = dbBlock.result ?? undefined;
+    const block = {
+      parent_index_block_hash: blockRes ? blockRes.index_block_hash : '0x00',
+      parent_block_hash: blockRes ? blockRes.block_hash : "0x00",
+      parent_microblock_hash: '',
+      parent_microblock_sequence: 0,
+      block_height: blockRes ? blockRes.block_height + 1 : 1,
+    };
+
+    const tx = {
+      tx_id: '0x1234',
+      tx_index: 4,
+      anchor_mode: AnchorMode.Any,
+      type_id: DbTxTypeId.Coinbase,
+      raw_result: '0x0100000000000000000000000000000001', // u1
+      canonical: true,
+      microblock_canonical: true,
+      microblock_sequence: I32_MAX,
+      microblock_hash: '',
+      parent_index_block_hash: block.parent_microblock_hash,
+      parent_block_hash: block.parent_index_block_hash,
+      post_conditions: Buffer.from([0x01, 0xf5]),
+      fee_rate: 1234n,
+      sender_address: 'sender-addr',
+    };
+
+    const dbMinerReward1 = {
+      canonical: true,
+    };
+
+    const dbMinerReward2 = {
+      canonical: true,
+      recipient: 'testAddr2',
+    };
+
+    const data = new TestBlockBuilder(block).addTx(tx).addMinerReward(dbMinerReward1).addMinerReward(dbMinerReward2).build();
```

Comment:
Place each of these in a separate line:
```suggestion
    const data = new TestBlockBuilder(block)
      .addTx(tx)
      .addMinerReward(dbMinerReward1)
      .addMinerReward(dbMinerReward2)
      .build();
```